invenio_app_ils/records/api.py

```diff
@@ -553,6 +553,14 @@ def dumps(self):
             "email": self.email,
         }
 
+    def dumps_loader(self, include_keys=None):
```

Comment:
I would change the `include_keys` for `**kwargs`, so it can be extensible if needed.